### PR TITLE
SUIT-13277 consolidate texts from fe and js api

### PR DESCRIPTION
**Pull request by andriichumak**

### CODE REVIEW

**Comment by andriichumak** [comment]
You're right, updated. I'm still looking into other branches here, will probably merge those first and fix conflicts here, so please do not merge it just yet

**Comment by andriichumak** [comment]
Please, review. @givehug @JeKachan @AntonSBilan

File: packages/translate/lib/__tests__/progress.test.ts [hunk 1]
```diff
@@ -1,4 +1,5 @@
-import {PROGRESS_STATUS, translateProgress} from '../progress';
+import {PROGRESS_STATUS} from "../constants";
```
Comment (JeKachan) [inline packages/translate/lib/__tests__/progress.test.ts]: double qoutes

Comment (andriichumak) [reply packages/translate/lib/__tests__/progress.test.ts]: ups, fixed

**Pull request merged**